app/views/devise/invitations/edit.html.erb

```diff
@@ -31,7 +31,7 @@
     <div class="registration__field">
       <%= f.form_row :password_confirmation do %>
         <%= f.errors_for :password_confirmation %>
-        <%= f.label :password_confirmation, class: "form__label-heading" %>
+        <%= f.label :password_confirmation, t('authentication.accept_invitation_page.field_labels.password_confirmation'), class: "form__label-heading" %>
```

Comment:
Thanks, @asmega. Not too sure I follow though. I'm explicitly calling a key here because it is being named differently to the normal password_confirmation on the model (i.e. this is "Confirm new password", not "Confirm password"). Or am I grabbing the wrong end of the stick? :)